### PR TITLE
next: rollout 32.20200615.1.3 barrier release

### PR DESCRIPTION
**Pull request by dustymabe**
See the following issue for context:
https://github.com/coreos/fedora-coreos-tracker/issues/484#issuecomment-649009580

### CODE REVIEW

File: updates/next.json [hunk 1]
```diff
@@ -1,16 +1,24 @@
...
+    },
+    {
+      "version": "32.20200615.1.3",
       "metadata": {
         "rollout": {
-          "start_epoch": 1592859600,
+          "start_epoch": 1593100800,
           "start_percentage": 0.0,
-          "duration_minutes": 2880
+          "duration_minutes": 1440
```
Comment (dustymabe) [inline updates/next.json:21]: should we make this less  than a day so we can start the next rollout tomorrow morning (US eastern time)?

File: updates/next.json [hunk 2]
```diff
@@ -1,16 +1,24 @@
...
   "metadata": {
-    "last-modified": "2020-06-22T18:38:34Z"
+    "last-modified": "2020-06-25T15:18:18Z"
   },
   "releases": [
     {
-      "version": "32.20200615.1.2",
+      "version": "32.20200615.1.3",
+      "metadata": {
+        "barrier": {
```
Comment (jlebon) [inline updates/next.json:10]: Hmm, shouldn't this barrier be in the same array element as the rollout?

File: updates/next.json (hunk 1 above)
Comment (jlebon) [reply updates/next.json:21]: Hmm, yeah sounds good to me.

File: updates/next.json (hunk 2 above)
Comment (dustymabe) [reply updates/next.json:10]: @lucab - can you say?

Comment (lucab) [reply updates/next.json:10]: Yes @jlebon is right, versions are unique keys, see https://github.com/coreos/fedora-coreos-tracker/blob/master/metadata/updates/specifications.md.

Comment (dustymabe) [reply updates/next.json:10]: ok I think I fixed it

**Pull request merged**